test/lib/ufe/testRotatePivot.py

```diff
@@ -158,12 +158,14 @@ def testRotatePivotCmd(self):
         rot = om.MEulerRotation(0, 0, rotZ)
         # Pivot around x=0.
         pivot = om.MPoint(-10, 0, 0)
+        pivotTranslate = om.MPoint(0, 0, 0)
         xyWorldValue = sin(rotZ) * 10
+        rotatedPosition = [xyWorldValue, xyWorldValue, 0]
 
         # USD sphere is at (10, 0, 0) in local space, and since its parents
         # have an identity transform, in world space as well.
         spherePath = ufe.PathString.path(
-                '|usdSphereParent|usdSphereParentShape,/sphereXform/sphere')
+                '|usdSphereParent|usdSphereParentShape,/sphereXform')
```

Comment:
This is a minor change but it means that all the work is done on /sphereXform, which already has some translation. Keeping all the transform stack together makes sense to me.